Incompatible with django-querysetsequence==0.9

```
project/app/tests/test_autocomplete.py:42: in test_autocomplete_endpoints_access                                                                                                          
    response = p.callback(fake_request)                                                                                                                                                       
.venv/lib/python3.6/site-packages/django/views/generic/base.py:69: in view                                                                                                                    
    return self.dispatch(request, *args, **kwargs)                                                                                                                                            
project/app/dal.py:21: in dispatch                                                                                                                                                        
    return super().dispatch(request, *args, **kwargs)                                                                                                                                         
.venv/lib/python3.6/site-packages/dal/views.py:49: in dispatch                                                                                                                                
    return super(ViewMixin, self).dispatch(request, *args, **kwargs)                                                                                                                          
.venv/lib/python3.6/site-packages/django/views/generic/base.py:89: in dispatch                                                                                                                
    return handler(request, *args, **kwargs)                                                                                                                                                  
.venv/lib/python3.6/site-packages/django/views/generic/list.py:142: in get                                                                                                                    
    self.object_list = self.get_queryset()                                                                                                                                                    
project/app/dal.py:60: in get_queryset                                                                                                                                                    
    qs = self.mixup_querysets(qs)                                                                                                                                                             
.venv/lib/python3.6/site-packages/dal_queryset_sequence/views.py:34: in mixup_querysets                                                                                                       
    if len(list(qs.query._querysets)):                                                                                                                                                        
E   AttributeError: 'QuerySetSequence' object has no attribute 'query'                                                                                                                        
```

Bisected to https://github.com/percipient/django-querysetsequence/commit/ff622bf3b049cb9c7413e8604223f686ff6c019e.

Removed `.query` here: https://github.com/percipient/django-querysetsequence/commit/ff622bf3b049cb9c7413e8604223f686ff6c019e#diff-f45794d6f38e102567b2d30bce0797d7L478

--- a/src/dal_queryset_sequence/fields.py
+++ b/src/dal_queryset_sequence/fields.py
@@ -19,7 +19,7 @@ class QuerySetSequenceFieldMixin(object):
         """Return the QuerySet from the QuerySetSequence for a ctype."""
         content_type = ContentType.objects.get_for_id(content_type_id)
 
-        for queryset in self.queryset.query._querysets:
+        for queryset in self.queryset.get_querysets():
             if queryset.model.__name__ == 'QuerySequenceModel':
                 # django-queryset-sequence 0.7 support dynamically created
                 # QuerySequenceModel which replaces the original model when it

--- a/src/dal_queryset_sequence/views.py
+++ b/src/dal_queryset_sequence/views.py
@@ -31,10 +31,11 @@ class BaseQuerySetSequenceView(BaseQuerySetView):
 
     def mixup_querysets(self, qs):
         """Return a queryset with different model types."""
-        if len(list(qs.query._querysets)):
-            limit = int(self.paginate_by / len(qs.query._querysets))
-            qs.query._querysets[0][:2]
-            qs = QuerySetSequence(*[q[:limit] for q in qs.query._querysets])
+        querysets = list(qs.get_querysets())
+        queryset_count = len(querysets)
+        if queryset_count:
+            limit = int(self.paginate_by / queryset_count)
+            qs = QuerySetSequence(*[q[:limit] for q in querysets])
         return qs
 
     def get_queryset(self):

